.github/workflows/ci.yml

```diff
@@ -87,11 +87,15 @@ jobs:
         with:
           node-version: ${{ matrix.node }}
       - name: install pnpm
+        if: matrix.node != 14
         shell: bash
         run: |
           PNPM_VER=$(jq -r '.packageManager | if .[0:5] == "pnpm@" then .[5:] else "packageManager in package.json does not start with pnpm@\n" | halt_error(1)  end' package.json)
           echo installing pnpm version $PNPM_VER
           npm i -g pnpm@$PNPM_VER
+      - name: install legacy pnpm for node14
```

Comment:
you could just leave `packageManager` as `7.32.0` in `package.json` and then you wouldn't need to do any of this. that's much simpler. people can still have pnpm 8 in their local installs, which is what's really important

Reply:
the goal is that everyone uses the same major version of the same pm. differences between pnpm 7 and 8 make it undesirable to allow contributing with both, as evidenced by the current problem of file: resolution in this PRs test run